index.html

```diff
@@ -184,59 +212,69 @@ <h2 class="workheading">Multi-Post Stories</h2>
             </div>
             
             <div>
-                <ul class="languagesFlexContainer">
+                <ul class="languagesFlexContainerReverse">
                     <li class="languages">CSS</li>
                     <li class="languages">HTML</li>
                     <li class="languages">Bootstrap</li>
                     <li class="languagestwo">Ruby</li>
                 </ul>
             </div>
-            <button type="button" class="projectButton">See Project</button>
+            <div class="button-box">
+                <button type="button" class="projectButton">See project</button>
+            </div>
+        </div>
+    </section>
+
+
+    <footer class="footer-form">
+        <div class="footer-heading">
+            <h2 class="contact-heading">CONTACT US</h2>
+            <p class="contact-details">
+                If you have an application you are
+                interested in developing, a feature that you
+                need built or a project that needs coding.
+                I’d love to help with it.
+            </p>
+        </div>
+        <form class="footer-input-form" action="https://formspree.io/f/mqkjaegz" method="post">
+            <ul class="footer-input">
+                <li>
+                    <input name="Fullname"  class="item1" type="text" maxlength="30" placeholder="Full name" required>
+                </li>
+                <li>
+                    <input name="Email" class="item2" type="email" placeholder="Email address" required>
+                </li>
+                <li>
+                    <textarea name="message" class="item3" required maxlength="500" placeholder="Enter text here"></textarea>
+                </li>
+                
+            </ul>
+            
+            <div class="footerButtonBox">
+                <button type="submit" class="footerButton">Send</button>
+            </div>
+            
+        </form>
+
+        <div class="horizontal-line-footer">
+
         </div>
 
-        <footer class="footer-form">
-            <div class="footer-heading">
-                <h2 class="contact-heading">CONTACT US</h2>
-                <p class="contact-details">
-                    If you have an application you are<br>
-                    interested in developing, a feature that you<br>
-                    need built or a project that needs coding.<br>
-                    I’d love to help with it.
-                </p>
-            </div>
```

Comment:
- [ ] `required`
- [ ] The contact forms for the desktop and mobile have different welcoming intro message the desktop `says` lets chat and mobile says contact us. please screenshot below

`your template`
![image](https://user-images.githubusercontent.com/66230306/186733772-2208965a-d0aa-455c-a351-b5e43d545d9e.png)

`figma template`
![lets chat](https://user-images.githubusercontent.com/66230306/186734532-ad4d1c9f-cb8b-463a-a436-36f4b89f1d60.png)

- [ ]  please make amends